### PR TITLE
Adapt URI to ORS production settings without default /ors base path

### PR DESCRIPTION
**Pull request by TheGreatRefrigerator**
## Issue

https://github.com/VROOM-Project/vroom/issues/1036

## Tasks

 - [x] Update `CHANGELOG.md` (remove if irrelevant)
 - [x] review

### CODE REVIEW

File: src/routing/ors_wrapper.cpp [hunk 1]
```diff
@@ -49,7 +49,7 @@ std::string OrsWrapper::build_query(const std::vector<Location>& locations,
 
   // Building query for ORS
   std::string query =
-    "POST /" + _server.path + "ors/v2/" + service + "/" + profile;
+    "POST /" + _server.path + "v2/" + service + "/" + profile;
```
[the remark this answers is not in the document]
Comment (TheGreatRefrigerator) [reply src/routing/ors_wrapper.cpp]: Done

File: CHANGELOG.md [hunk 2]
```diff
@@ -24,6 +24,7 @@
 - Support for URL path in host (#966)
 
 ### Changed
+- **Breaking:** default ors base path to be more dynamic ([#1036](https://github.com/VROOM-Project/vroom/issues/1036))
```
Comment (nilsnolde) [inline CHANGELOG.md:27]: Nit: I’d actually spell out that the hard coded /ors/v2 was removed and needs to be provided by in the argument. Saves a round trip to the PR to anyone reading this.

Comment (TheGreatRefrigerator) [reply CHANGELOG.md]: Added this and moved it to ### Removed (more fitting)

Comment (jcoupey) [reply CHANGELOG.md]: I'll just slightly rephrase that upon merging.

**Review by jcoupey — APPROVED** [review]

**Pull request merged**